Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,55 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+
+        StartOver:
+            InteractionManager.PerformInteractions();
+
+            Console.WriteLine("Whould you like to start again?");
+            var answer = Console.ReadLine();
+            if (answer.ToLower() == "y") goto StartOver;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this use of 'goto'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4896164)